frontend/src/metabase/lib/formatting/currency.js

```diff
@@ -65,3 +65,22 @@ export function getCurrencyOptions() {
     value: currency.code,
   }));
 }
+
+export function sortCurrencyOptionsByPriority(currencyOptions, key = "label") {
```

Comment:
I'm not sure if this function needs to be exposed.
How about:
- we make it local (no `export`)
- we use it in `getCurrencyOptions`
- we make sure `getCurrencyOptions` is used in all 3 currency pickers
- we drop the `key` argument

This way:
- we'll only have 1 import from `cljs/metabase.util.currency` in the project
- components will not care about the ordering, it will already come correct from `getCurrencyOptions`